agent_application/docker/docker-compose.yml

```diff
@@ -27,12 +27,16 @@ services:
     depends_on:
       - cqrs-postgres-db
   ssi-agent:
-    image: ssi-agent
-    restart: always
+    #image: ssi-agent
+    build:
+      context: ../..
+      dockerfile: ./agent_application/docker/Dockerfile
+    #restart: always
```

Comment:
Can we switch to something such as `restart: "${DOCKER_RESTART_POLICY:always}"`, so we don't have to comment it out for dev/prod environments?

Reply:
I added it, not sure if I like it. I want it to crash if something goes wrong.

Reply:
Okay, fair. I'm just not the biggest fan of having to comment in/out in different files (and always have to remember to set it back before I make a commit), but rather set it through some env variable that I can define for myself and that is untracked.